tests/it/docs.d

```diff
@@ -0,0 +1,201 @@
+module it.docs;
+
+import it;
+import dpp.translation;
+import unit_threaded: shouldEqual;
+
+import std.file: readText;
+import std.string: strip, indexOf;
+
+// TODO find a way to test text equality
```

Comment:
The tests currently fail. I cannot make the unit tests to print the result of the readFile.

Reply:
Fixed with `normalizeText` function